pkg/trainer/replicas.go

```diff
@@ -88,7 +94,67 @@ func (s *TFReplicaSet) Labels() KubernetesLabels {
 		"runtime_id": s.Job.job.Spec.RuntimeId})
 }
 
-func (s *TFReplicaSet) Create() error {
+// Transforms the tfconfig to work with grpc_tensorflow_server
+func transformClusterSpecForDefaultPS(clusterSpec ClusterSpec) string {
+
+	// sort by keys to make unit testing easier
+	keys := []string{}
+	for k := range clusterSpec {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+
+	var buf bytes.Buffer
+	isFirstJob := true
+	for _, k := range keys {
+		if !isFirstJob {
+			//separator between different job kinds
+			buf.WriteString(",")
+		}
+		isFirstJob = false
+		buf.WriteString(k)
+		//separator between job name and it's element
+		buf.WriteString("|")
+		isFirstElement := true
+		for _, e := range clusterSpec[k] {
+			if !isFirstElement {
+				//separator between different elements with same job type
+				buf.WriteString(";")
+			}
+			isFirstElement = false
+			buf.WriteString(e)
+		}
+	}
+	return buf.String()
+}
+
+func (s *TFReplicaSet) Create(config *spec.ControllerConfig) error {
+	if s.Spec.IsDefaultPS {
+		// Create the ConfigMap containing the sources for the default Parameter Server
+		err, cm := s.getDefaultPSConfigMap(config)
+		if err != nil {
+			log.Errorf("Error building PS ConfigMap: %v", err)
+			return err
+		}
+		_, err = s.ClientSet.CoreV1().ConfigMaps(NAMESPACE).Create(cm)
```

Comment:
Should we be using the namespace of the job as opposed to the constant NAMESPACE?
I think #39 added support for creating TfJobs in other namespaces.
It looks like all the other resources use s.Job.job.Metadata.Namespace for the namespace

Can you delete the constant "NAMESPACE" in training.go since it should be unused?